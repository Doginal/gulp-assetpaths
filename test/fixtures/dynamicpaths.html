<img src="<?php echo $imagePath ?>" />
<img src="<%= obj.imagePath %>" />
<script src="<% scriptname %>" />
<img style="display:none;" src="<%= obj.imagePath %>" />
background: url(<% imagepath %>)
background-image : url(<%= imagepath %>)
<a href="<%= assetpathYo %>" download="<%= assetname %>" />
$pattern = '/src="(.*js)"/';
$replacement = 'src="restylegc-js.php?$1"';
<a href="https://www.oldDomain.com/<?php echo $test; ?>">TEST</a>
<a href="/cart_checkout.php"><img src="/images/navigation_cart.png" alt="Shopping cart" width="30" height="33" />
<link rel="stylesheet" href="/css/pages/products/<?php echo $thisPage; ?>.css" type="text/css" />