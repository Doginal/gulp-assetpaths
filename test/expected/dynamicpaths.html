<img src="https://www.newDomain.com/<?php echo $imagePath ?>" />
<img src="https://www.newDomain.com/<%= obj.imagePath %>" />
<script src="https://www.newDomain.com/<% scriptname %>" />
<img style="display:none;" src="https://www.newDomain.com/<%= obj.imagePath %>" />
background: url(https://www.newDomain.com/<% imagepath %>)
background-image : url(https://www.newDomain.com/<%= imagepath %>)
<a href="https://www.newDomain.com/<%= assetpathYo %>" download="<%= assetname %>" />
$pattern = '/src="(.*js)"/';
$replacement = 'src="restylegc-js.php?$1"';
<a href="https://www.oldDomain.com/<?php echo $test; ?>">TEST</a>
<a href="/cart_checkout.php"><img src="https://www.newDomain.com/images/navigation_cart.png" alt="Shopping cart" width="30" height="33" />